pkg/discovery/servicehandler.go

```diff
@@ -92,16 +84,13 @@ func (s *serviceHandler) ToServiceDetails(asset *anypoint.Asset) []*ServiceDetai
 
 // getServiceDetail gets the ServiceDetail for the API asset.
 func (s *serviceHandler) getServiceDetail(asset *anypoint.Asset, api *anypoint.API) (*ServiceDetail, error) {
+	api.ActiveContractsCount = 0
```

Comment:
ActiveContractsCount will increment each time a contract is made. Without this, we will get a new revision for each contract. Setting this to 0 will stop the duplicate revisions.